announce/server.py

```
@@ -66,6 +66,15 @@ def get_group(groupid, Group, Cred, Member):
     group = session.query(Group).filter_by(id=groupid).first()
     return render("group.html", group=group)
 
+@app.get("/delgroup", name="del_group")
+@fill_args
+def del_group(groupid,Group):
+    session = bottle.request.session
+    x = session.query(Group).get(groupid)
```

If the group does not exist can we raise a 404 error here please? I usually do 

```python
g = session.query(Group).filter_by(id=groupid).first()
if g is None:
    return bottle.abort(404, 'no such group')
```